js/copy-service/Formatting/Formatting.test.ts

```diff
@@ -9,7 +9,7 @@ describe('Formatting', () => {
     test('sets valid options to the instance', () => {
       const options = {
         sibling: new Formatting({}),
-        copy: new Verbatim({ text: 'some copy' }),
+        copy: new Verbatim({ sibling: null, text: 'some copy' }),
```

Comment:
NP: Here and elsewhere:
```suggestion
        copy: new Verbatim({ 
          sibling: null, 
          text: 'some copy' 
        }),
```